docs/content/theme/_2_theme-palette.md

```diff
@@ -0,0 +1,134 @@
+
++++
+title = "Theme palette"
+description = ""
+weight = 2
+draft = false
+bref = ""
+toc = true
+menu = "theme"
+tags = [ "theme" ]
++++
+
+
+## List of theme palette variables
+
+These variables are global hooks to override colors, fonts, spacing and more throughout the library of web components. Generally this is all you need, though component specific variables are also available. Feel free to copy this list and define the values needed for your site or application. Please note that the logo URL paths are empty here.
+
+```
+:root {
```

Comment:
Would this be helpful to print as a table?


| Variable name | Type       | Project default |
| ------------- | ---------- | --- |
| `--pfe-theme--color--ui-base` | Color | #0076e0 |

Reply:
I didn't hear back on this so I added a commit with the update to use a table and to show the current values in the PFE system instead of the dotcom colors.